python/pyspark/sql/functions.py

```diff
@@ -411,7 +415,7 @@ def monotonically_increasing_id():
 
     The generated ID is guaranteed to be monotonically increasing and unique, but not consecutive.
     The current implementation puts the partition ID in the upper 31 bits, and the record number
-    within each partition in the lower 33 bits. The assumption is that the data frame has
+    within each partition in the lower 33 bits. The assumption is that the DataFrame has
```

Comment:
It'd be nicer if we use `:class:`DataFrame` instead of the plain string `DateFrame` if you would like to change this. This makes this as a link to the `DataFrame` class.